addons/website_sale/static/src/js/tours/tour_utils.js

```diff
@@ -14,7 +14,28 @@ odoo.define("website_sale.tour_utils", function (require) {
         };
     }
 
+    function selectPriceList(pricelist) {
+        return [
+            {
+                content: "Click on pricelist dropdown",
+                trigger: "div.o_pricelist_dropdown a[data-toggle=dropdown]",
+            },
+            {
+                content: "Click on pricelist",
+                trigger: `span:contains(${pricelist})`,
+            },
+        ]
+    }
+    function assertProductPrice(attribute, value, productName) {
```

Comment:
I like these utils function thanks.
It would be beneficial to add a note to this function that it only works for the product grid though 🙂 .